src/setupTest.js

```
@@ -27,9 +27,11 @@ import { appendBrowserTimezoneToUrl, executeThunk } from './utils';
 import buildSimpleCourseAndSequenceMetadata from './courseware/data/__factories__/sequenceMetadata.factory';
 
 class MockLoggingService {
-  logInfo = jest.fn();
+  // eslint-disable-next-line no-console
```

Mocking the logger causes the tests to swallow any errors that are logged in the run. Is this reasonable to leave in?

---

Sure? Seems reasonable.